src/index.js

```diff
@@ -0,0 +1,13 @@
+import _ from 'lodash';
+import './style.css';
+
+function component() {
+  const element = document.createElement('div');
+
+  // Lodash, now imported by this script
+  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+
```

Comment:
# Optional:
- I would advise you to remove this line of codes, and also remove `import _ from 'lodash'` because it is affecting your beautiful frame.
![Capture](https://user-images.githubusercontent.com/63972670/217207928-b9472b3f-4e04-4035-92ac-5fed1210d831.PNG)

Reply:
@OLIPLICHE Thanks for pointing out that issue. I will remove it.